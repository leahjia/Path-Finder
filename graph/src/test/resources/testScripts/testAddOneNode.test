# DesignMap adding nodes testing file.
# Tests that a single node can be added to the map

# Create a graph
CreateGraph map1

# Add one node to map1
AddNode map1 n1