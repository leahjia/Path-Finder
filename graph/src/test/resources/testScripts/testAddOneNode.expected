# DesignMap adding nodes testing file.
# Tests that a single node can be added to the map

# Create a graph
created graph map1

# Add one node to map1
added node n1 to map1
